src/panels/lovelace/editor/config-elements/hui-map-card-editor.ts

```diff
@@ -136,7 +136,7 @@ export class HuiMapCardEditor extends LitElement implements LovelaceCardEditor {
   }
 
   get _geo_location_sources(): string[] {
-    return this._config!.geo_location_sources || [];
```

Comment:
Won't this cause `[object Object]` as they may not be strings anymore?

Reply:
If it was not a string this editor would not be shown, as it would fail the assert in setConfig.

Reply:
stale